### PR TITLE
Add a preact example.

### PR DESCRIPTION
**Pull request by arunoda**
Added an  example using `preact-compat`.
Here's the live version: https://hello-world-nrqihmdggp.now.sh/

### CODE REVIEW

File: examples/using-preact/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+# Hello World example
+
+## How to use
+
+Download the example (or clone the repo)[https://github.com/zeit/next.js.git]:
```
Comment (rauchg) [inline examples/using-preact/README.md]: `.git` not needed here

Comment (arunoda) [inline examples/using-preact/README.md]: I just copied this from the hello-world example. May be we need to change this for all examples. Shall we do it with another PR?

**Comment by rauchg** [comment]
Yes

**Comment by lukeed** [comment]
Oh man, I was waiting to open this same PR 😉

**Comment by rauchg** [comment]
Thanks @lukeed  ;)

**Comment by alexnm** [comment]
@arunoda, @rauchg 
 I just stumbled upon this and wanted to give it a try but it's throwing an error when navigating to `/about`

```
Uncaught (in promise) TypeError: Cannot read property '_pendingForceUpdate' of undefined
    at setPendingForceUpdate (index.js?0d84949:20)
    at traverseRenderedChildren (index.js?0d84949:6)
    at deepForceUpdate (index.js?0d84949:40)
    at AppContainer.componentWillReceiveProps (AppContainer.dev.js?0d84949:45)
    at callMethod (preact-compat.js?0c5b0f7:458)
    at AppContainer.componentWillReceiveProps (preact-compat.js?0c5b0f7:469)
    at setComponentProps (preact.js?9f0705e:276)
    at renderComponent (preact.js?9f0705e:310)
    at rerender (preact.js?9f0705e:72)
```

I wanted to test if this is actually loading `preact` instead of `react` and I was able to use `React.createClass` in one of the pages.

I haven't used `preact` before, maybe i'm missing something?

**Comment by developit** [comment]
What libary is providing that `deepForceUpdate()` method?

**Comment by rauchg** [comment]
One by @gaearon IIRC

**Comment by developit** [comment]
Ahh perhaps React Hot Loader.

**Comment by craigsssmith** [comment]
I'm having a little difficulty using Preact with my next.js app.

```
Uncaught TypeError: Cannot set property createElement of #<Object> which has only a getter
```

I think the problem is related to React Hot Loader, with the error being thrown here: https://github.com/gaearon/react-hot-loader/blob/next/src/patch.dev.js#L166

Have you come across this before?

**Comment by developit** [comment]
hmm - perhaps enable `loose` mode for the `es2015` preset? It seems like this is tripping over frozen es2015 module exports.

**Comment by craigsssmith** [comment]
That's exactly what I thought at first, but it doesn't make any difference. Apologies, I should have mentioned that I'd tried that already. :-)
